db/migrate/20150102201942_create_votes.rb

```diff
@@ -0,0 +1,10 @@
+class CreateVotes < ActiveRecord::Migration
+  def change
+    create_table :votes do |t|
+      t.integer :user_id, null: false
+      t.integer :review_id, null: false
```

Comment:
Add an index on the foreign keys `user_id` and `review_id`. Also, if you want to ensure the uniqueness of those columns you can index both of them:

```ruby
add_index :votes, [:user_id, :review_id], unique: true
# Still need to add an index on review_id so you can find all votes for a review quickly
add_index :votes, :review_id
```